src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/LanguageServices/CSharpSyntaxGeneratorInternal.cs

```diff
@@ -95,11 +95,11 @@ internal override SyntaxNode InterpolatedStringExpression(SyntaxToken startToken
         internal override SyntaxNode InterpolatedStringText(SyntaxToken textToken)
             => SyntaxFactory.InterpolatedStringText(textToken);
 
-        internal override SyntaxToken InterpolatedStringTextToken(string content)
+        internal override SyntaxToken InterpolatedStringTextToken(string content, string value)
             => SyntaxFactory.Token(
                 SyntaxFactory.TriviaList(),
                 SyntaxKind.InterpolatedStringTextToken,
-                content, "",
+                content, value,
```

Comment:
Does it matter? I thought the Value and ValueText would be fixed on its own when the text is re-parsed.

Reply:
New test framework verifies that the refactoring produces the same tree as the compiler, with the exception of trivia associativity.

Reply:
this is also valuable since CodeActions pass their trees to multiple cleanup passes.  We want the trees we produce to be as properly-shaped as possible if those later passes examine this data.  Ideally they woudl be identical to what the parser produces.  But i know we don't 100% meet that currently (unfortunately).